.github/workflows/renovate-auto-merge.yml

```diff
@@ -4,11 +4,14 @@ on:
     types: [ opened, synchronize, reopened ]
 jobs:
   all-checks-passed:
-    if: contains(github.event.pull_request.labels.*.name, 'automerge') && github.actor == 'renovate[bot]'
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'automerge') && github.actor == 'renovate[bot]'}}
     name: All checks passed
     runs-on: ubuntu-latest
     permissions:
       checks: read
       contents: read
+    environment: delayenv
```

Comment:
Run will be delayed for 180 minutes. I don't know if this is enough to run our whole CI. However, we can tweak this.